ansible/tasks/phpmyadmin.yml

```diff
@@ -10,38 +15,58 @@
       - "php7.2-curl"
     state: "present"
 
-- name: "clone phpmyadmin repository"
-  git:
-    repo: "https://github.com/phpmyadmin/phpmyadmin.git"
-    dest: "/var/www/phpmyadmin.{{ canonical_hostname }}"
-    depth: 1
-    force: true
-    version: "STABLE"
-  diff: false
-
-- name: "place phpmyadmin config.inc.php"
-  template:
-    src: "templates/var/www/phpmyadmin/{{ filename }}.j2"
-    dest: "/var/www/phpmyadmin.{{ canonical_hostname }}/{{ filename }}"
-    mode: "600"
-  vars:
-    filename: "config.inc.php"
-
-- name: "install phpmyadmin composer dependencies"
-  composer:
-    working_dir: "/var/www/phpmyadmin.{{ canonical_hostname }}"
-    no_dev: true
-    command: "update"
-
-- name: "set permissions on phpmyadmin webroot"
+- name: "create phpmyadmin user"
+  user:
+    name: "phpmyadmin"
+    home: "/var/www/phpmyadmin"
+    shell: "/bin/bash"
+    system: true
+    state: "present"
+
+- name: "create phpmyadmin webroot"
   file:
-    path: "/var/www/phpmyadmin.{{ canonical_hostname }}"
-    recurse: true
+    path: "/var/www/phpmyadmin/phpmyadmin.{{ canonical_hostname }}"
     owner: "www-data"
-    group: "www-data"
+    group: "phpmyadmin"
+    mode: "2775"
     state: "directory"
 
+- block:
+    - name: "clone phpmyadmin repository"
+      git:
+        repo: "https://github.com/phpmyadmin/phpmyadmin.git"
```

Comment:
Do we still need to do this clone or is there a package that we can use instead? (Maybe we can remove the composer dependency as well)

Reply:
There is an apt package in the bionic repo's now, but I think I tried it during #179 and ran into problems getting it to work. Don't remember what exactly, so it might benefit from another look. However, because of the earlier attempt I'd like to save that for another time.